lib/scaleway/charts/q-application/templates/deployment.j2.yaml

```diff
@@ -0,0 +1,89 @@
+{%- if not is_storage %}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ sanitized_name }}
+  namespace: {{ namespace }}
+  labels:
+    ownerId: {{ owner_id }}
+    envId: {{ environment_id }}
+    appId: {{ id }}
+    app: {{ sanitized_name }}
+  annotations:
+    releaseTime: {% raw %}{{ dateInZone "2006-01-02 15:04:05Z" (now) "UTC"| quote }}{% endraw %}
+spec:
+  replicas: {{ total_instances }}
+  strategy:
+    type: RollingUpdate
+    {% if total_instances == 1 %}
+    rollingUpdate:
+      maxSurge: 1
+    {% endif %}
+  selector:
+    matchLabels:
+      ownerId: {{ owner_id }}
+      envId: {{ environment_id }}
+      appId: {{ id }}
```

Comment:
OPT: only matching on the appId should be enough

Reply:
Done, removed app name